### PR TITLE
Init Payment Delegation doc

### PR DESCRIPTION
**Pull request by spacesailor24**
Adds a simple overview of all the functions currently available for Payment Delegation Database contract

Closes: [LIT-3230](https://linear.app/litprotocol/issue/LIT-3230/initial-docs-for-delegation-database-contract)

### CODE REVIEW

File: docs/sdk/authentication/payment-delegation-database.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+### `undelegatePaymentsBatch`
+
+[Link to implementation](https://github.com/LIT-Protocol/lit-assets/blob/f73b7e2a3c982fa8fe3a13ed816e07fa6e04fbd8/blockchain/contracts/contracts/lit-node/PaymentDelegation/PaymentDelegationFacet.sol#L89-L98)
+
+Does the same as [undelegatepayments](#undelegatepayments), but accepts a list of `user`s as input.
+
+```js
+function undelegatePaymentsBatch(address[] memory users) public
+```
```
Comment (joshLong145) [inline docs/sdk/authentication/payment-delegation-database.md:143]: In general this looks good as an `api doc` but we will need supporting examples of how users will get 'delegation signatures ` for example as from this api doc it is non obvious.

File: docs/sdk/authentication/payment-delegation-database.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+- [`getRestriction`](#getrestriction)
+- [`delegatePaymentsBatch`](#delegatepaymentsbatch)
+- [`undelegatePaymentsBatch`](#undelegatepaymentsbatch)
+
+In order to start using the Payment Database, you should fetch the address of the latest implementation:
+
+```js
+const paymentDelegationFacet = await ethers.getContractAt(
+    'PaymentDelegationFacet',
+    await paymentDelegationDiamond.getAddress()
```
Comment (joshLong145) [inline docs/sdk/authentication/payment-delegation-database.md:33]: how does the user get this `paymentDelegationDiamond.getAddress()' as this looks to be taken from a test case.

**Comment by alexvotofuture** [comment]
Note this may change once our work on the delegation server finishes this week.

File: docs/sdk/authentication/payment-delegation-database.md (hunk 2 above)
Comment (glitch003) [reply docs/sdk/authentication/payment-delegation-database.md:33]: https://github.com/LIT-Protocol/networks it's in this repo, or they can use the resolver etc

File: docs/sdk/authentication/payment-delegation-database.md (hunk 1 above)
Comment (glitch003) [reply docs/sdk/authentication/payment-delegation-database.md:143]: sorry what do you mean by delegation signatures?  i don't think the payment delegation contract system needs those.

**Review by glitch003 — APPROVED** [review]

**Pull request closed without merging**